src/sweetalert2.js

```diff
@@ -894,6 +894,15 @@ function modalDependant() {
 
     // fix scroll
     sweetContainer.scrollTop = 0;
+
+    // Observe changes inside the modal and adjust height
+    if (typeof MutationObserver !== 'undefined') {
```

Comment:
this change is causing duplicating of this handler. Each modal will add a handler without removing previous.